### ISSUE TITLE
Allow a teacher or admin to create a new lesson

### ISSUE THREAD

**Issue by amellie** [Ali-Jahankah/bugsquashers-edu-app#16]
## Dependency

Depends on #14 

## Story

**As a** Teacher or an Admin
**I Want** to create a new lesson to an **existing** module
**And** upload the lesson document
**So That** the lesson is grouped into a module
**And** my students will be able to view them

## Acceptance Criteria
- For now, you don't need to implement the button to go to this Create page (this will be handled in a separate task). Just imagine the user already clicks a link/button to reach here. E.g. `url.com/lesson/create?id=1&module_id=2`. Here, we can see that the page should show the create page for lesson with ID = 1, which will be linked to module ID 2
- The form should show a dropdown containing all modules (but it is selected by default to the module in the URL, in the above example it is module with ID 2), lesson name, description, and document to upload. All of these are mandatory (except for description)
- Only admins and teachers can access the page
- Created date is submission date
- Type: Attachment
